src/css/base/form.scss

```diff
@@ -11,14 +12,73 @@ legend {
 
 label {
   margin-top: 0;
+  font-weight: 600;
 }
 
 input[type="email"] {
-  font-size: 1rem;
   padding: 0.7rem 0.7rem;
 }
 
 form button[type="submit"],
 form input[type="submit"] {
   margin-top: 1.5rem;
 }
+
+input, 
+input[type="text"], 
+input[type="email"], 
+input[type="password"], 
+input[type="url"], 
+input[type="tel"], 
+input[type="number"], 
+input[type="search"], 
+input[type="file"], 
+input[type="date"], 
+input[type="datetime-local"], 
+input[type="month"], 
+input[type="time"], 
+input[type="week"], 
+textarea, 
+select {
+  font-size: 1rem;  
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+textarea {
+  line-height: 1.45;
+}
+
+select {
+  background-size: 1rem;
+}
+
+fieldset div {
+  padding-top: .75rem;
+  padding-bottom: .25rem;
+}
+
+button.button-cancel {
```

Comment:
Does this need to be qualified with `button` or can it just be `.button-cancel`?